assets/stylesheets/common/campaign.scss

```diff
@@ -131,7 +131,7 @@ body.archetype-regular {
     padding: 2em;
     background-color: var(--primary-very-low);
 
-    @media screen and (max-width: 400px) {
+    @media screen and (max-width: 700px) {
```

Comment:
Don't we have a function for this? like `@include breakpoint(name`

Reply:
Good catch! I updated all of the media queries to use the breakpoint mixin. https://github.com/discourse/discourse-subscriptions/pull/273/commits/1867f2ab4fcbacb26750f6e9b6f12003844828e7

The ` 700px` breakpoints now target the tablets which